install.sh

```diff
@@ -2,15 +2,18 @@
 
 set -x
 set -e
+
+namespace="servicecomb"
 kubectl create -f deploy/namespace.yaml
 
-bash -x deploy/signed-cert.sh --service sidecar-injector-webhook-mesher-svc --secret sidecar-injector-webhook-mesher-certs --namespace chassis
+bash -x deploy/signed-cert.sh --service sidecar-injector-webhook-mesher-svc \
+ --secret sidecar-injector-webhook-mesher-certs --namespace ${namespace}
 
-kubectl create -f deploy/mesherconfigmap.yaml -n chassis
-kubectl create -f deploy/configmap.yaml -n chassis
-kubectl create -f deploy/serviceaccount.yaml -n chassis
-kubectl create -f deploy/clusterrole.yaml -n chassis
-kubectl create -f deploy/clusterbinding.yaml -n chassis
-kubectl create -f deploy/deployment.yaml -n chassis
-kubectl create -f deploy/service.yaml -n chassis
-kubectl create -f deploy/mutatingwebhook.yaml -n chassis
+kubectl create -f deploy/mesherconfigmap.yaml -n ${namespace}
+kubectl create -f deploy/configmap.yaml -n ${namespace}
```

Comment:
${namespace} currently uses the namespace as **servicecomb** allways. 
Please use namespace as argument and pass to install.sh file and use passed namespace everywhere.

So If argument provided than use that name in namespace.yaml for creating..If not provided use **servicecomb**  as a namespace.

And Update in **README.md** and https://mesher.readthedocs.io/en/latest/

Reply:
later use helm template values @savitaashture